tests/helper/helper_podman.go

```diff
@@ -42,3 +45,14 @@ func ExtractK8sAndOcComponentsFromOutputOnPodman(out string) []string {
 
 	return handled
 }
+
+// Returns version of installed podman
+func GetPodmanVersion() string {
+	cmd := exec.Command("podman", "version", "--format", "json")
+	out, err := cmd.Output()
+	Expect(err).ToNot(HaveOccurred())
```

Comment:
Thanks for the changes. One last suggestion to help investigate if an error occurs (as we did elsewhere):
```suggestion
	Expect(err).ToNot(HaveOccurred(), func() string {
		if exiterr, ok := err.(*exec.ExitError); ok {
			err = fmt.Errorf("%s: %s", err, string(exiterr.Stderr))
		}
		return err.Error()
	})
```

Otherwise, it might return unhelpful output:

```
• [FAILED] [0.111 seconds]                                                                                                                                                                     
odo dev command tests when setting git config and running odo dev [BeforeEach] should create vcs-uri annotation for the deployment when running odo dev [podman]                               
  [BeforeEach] /home/asoro/work/projects/odo/tests/integration/cmd_dev_test.go:2374                                                                                                            
  [It] /home/asoro/work/projects/odo/tests/integration/cmd_dev_test.go:2388                                                                                                                    
                                                                                                                                                                                               
  [FAILED] Unexpected error:                                                                                                                                                                   
      <*exec.ExitError | 0xc0002085a0>: {                                                                                                                                                      
          ProcessState: {                                                                                                                                                                      
              pid: 3626644,                                                                                                                                                                    
              status: 32000,                                                                                                                                                                   
              rusage: {                                                                                                                                                                        
                  Utime: {Sec: 0, Usec: 14442},                                                                                                                                                
                  Stime: {Sec: 0, Usec: 22637},                                                                                                                                                
                  Maxrss: 59904,                                                                                                                                                               
                  Ixrss: 0,                                                                                                                                                                    
                  Idrss: 0,                                                                                                                                                                    
                  Isrss: 0,                                                                                                                                                                    
                  Minflt: 4579,                                                                                                                                                                
                  Majflt: 0,                                                                                                                                                                   
                  Nswap: 0,                                                                                                                                                                    
                  Inblock: 0,                                                                                                                                                                  
                  Oublock: 0,                                                                                                                                                                  
                  Msgsnd: 0,                                                                                                                                                                   
                  Msgrcv: 0,                                                                                                                                                                   
                  Nsignals: 0,                                                                                                                                                                 
                  Nvcsw: 321,                                                                                                                                                                  
                  Nivcsw: 1,                                                                                                                                                                   
              },                                                                                                                                                                               
          },                                                                                                                                                                                   
          Stderr: [69, 114, 114, 111, 114, 58, 32, 110, 97, 109, 101, 115, 112, 97, 99, 101, 115, 32, 97, 114, 101, 32, 110, 111, 116, 32, 115, 117, 112, 112, 111, 114, 116, 101, 100, 32, 98,
 121, 32, 116, 104, 105, 115, 32, 118, 101, 114, 115, 105, 111, 110, 32, 111, 102, 32, 76, 105, 98, 112, 111, 100, 44, 32, 112, 108, 101, 97, 115, 101, 32, 117, 110, 115, 101, 116, 32, 116, 1
04, 101, 32, 96, 110, 97, 109, 101, 115, 112, 97, 99, 101, 96, 32, 102, 105, 101, 108, 100, 32, 105, 110, 32, 99, 111, 110, 116, 97, 105, 110, 101, 114, 115, 46, 99, 111, 110, 102, 58, 32, 11
0, 111, 116, 32, 121, 101, 116, 32, 105, 109, 112, 108, 101, 109, 101, 110, 116, 101, 100, 10], 
      }
      exit status 125
  occurred
  In [BeforeEach] at: /home/asoro/work/projects/odo/tests/helper/helper_podman.go:54 @ 04/19/23 14:37:06.534
```